src/test/java/study/SetCollectionTest.java

```diff
@@ -0,0 +1,53 @@
+package study;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.*;
+
+@DisplayName("Set Collection에 대한 학습 테스트")
+public class SetCollectionTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Test
+    @DisplayName("요구사항 1 - size() 메소드를 활용해 Set의 크기 확인")
+    void size() {
+        // when
+        int size = numbers.size();
+
+        // then
+        assertThat(size).isEqualTo(3);
+    }
```

Comment:
```suggestion
    @Test
    @DisplayName("요구사항 1 - size() 메소드를 활용해 Set의 크기 확인")
    void size() {
        assertThat(numbers).hasSize(3);
    }
```
알고 계실 수 있지만 assertj 에서는 `Collection` 에 대한 검증 기능도 지원되고 있습니다!!
중요한 부분은 아니니 참고만해주세요 😆